README.md

```diff
@@ -42,12 +42,26 @@ After setting up the cluster, you can start the [client](https://github.com/haze
 
 You can launch Hazelcast Enterprise Docker Container by running the following command. You can find the full list of Hazelcast Enterprise versions to replace $HAZELCAST_VERSION at [Official Hazelcast Docker Hub](https://store.docker.com/community/images/hazelcast/hazelcast-enterprise/tags).
 
-please contact sales@hazelcast.com for trial license.
+Please request trial license [here](https://hazelcast.com/hazelcast-enterprise-download/) or contact sales@hazelcast.com.
 
 ```
-docker run -ti -e HZ_LICENSE_KEY=YOUR_LICENSE_KEY hazelcast/hazelcast-enterprise:$HAZELCAST_VERSION
+$ docker run -e HZ_LICENSE_KEY=<your_license_key> hazelcast/hazelcast-enterprise:$HAZELCAST_VERSION
 ```
 
+# Hazelcast Enterprise Hello World
+
+To run two Hazelcast nodes with Management Center, use the following commands.
+
+```
+$ docker run -p 8080:8080 --name mancenter hazelcast/management-center
+$ docker run -p 5701:5701 -e HZ_LICENSE_KEY=<your_license_key> -e MANCENTER_URL="http://mancenter:8080/hazelcast-mancenter" -e JAVA_OPTS="-Dhazelcast.local.publicAddress=192.168.1.1:5701" --link mancenter hazelcast/hazelcast-enterprise
```

Comment:
I would explain `MANCENTER_URL` env. in the README as it is default place to look up MC URL by docker image.

Reply:
Added. Thanks! PTAL.